model/time.go

```diff
@@ -181,7 +181,7 @@ func (d *Duration) Type() string {
 	return "duration"
 }
 
-var durationRE = regexp.MustCompile("^([0-9]+)(y|w|d|h|m|s|ms)$")
+var durationRE = regexp.MustCompile("^(([0-9]+)y)?(([0-9]+)w)?(([0-9]+)d)?(([0-9]+)h)?(([0-9]+)m)?(([0-9]+)s)?(([0-9]+)ms)?$")
```

Comment:
This doesn't match `0`

Reply:
Neither did the previous implementation. "0" is treated differently. Note that this is an unexported regex.

Reply:
PS: there is already a testcase for "0".